packages/@react-spectrum/overlays/src/Popover.tsx

```diff
@@ -96,12 +103,14 @@ const PopoverWrapper = forwardRef((props: PopoverWrapperProps, ref: RefObject<HT
     isNonModal,
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     isDismissable,
+    overlayProps,
+    preventScroll,
     ...otherProps
   } = props;
-  let {overlayProps} = useOverlay({...props, isDismissable: isDismissable && isOpen}, ref);
   let {modalProps} = useModal({
     isDisabled: isNonModal
   });
+  usePreventScroll({isDisabled: !preventScroll});
```

Comment:
Should/did we consider moving this behavior to `useOverlay` as a whole? Potentially something that aria hook users should have enabled by default for their own overlays

Reply:
I think that makes sense